src/pages/contracts/[id].vue

```diff
@@ -54,12 +66,21 @@ const { isLoading } = useLoading()
 
 await useAsyncData(() => fetchContractDetails(route.params.id))
 
-if (process.client) {
+const contractFound = ref(true)
```

Comment:
```suggestion
const isContractFound = ref(true)
```

similar to other occurences

Reply:
Thank you, fixed

Reply:
Thanks, looks better. If you want to make it even nicer, I suggest to sort the variables in [id] pages (optional just for consistency). They are in different order in almost every page